frontend/shared/utils/HAWCUtils.js

```diff
@@ -203,8 +203,22 @@ class HAWCUtils {
         return false;
     }
 
-    static urlify(str) {
-        return slugify(str, {remove: /[^\w\s-_]/g});
+    static randomString(length) {
```

Comment:
very similar code already exists here w/ a different implementation https://github.com/shapiromatron/hawc/blob/e54a8bee877e2cd63d03109106c3c108be3bf197/frontend/shared/utils/helpers.js#L246-L250

Reply:
A quick benchmark shows the newer implementation being a bit faster (I imagine since its not relying on a regex):
![image](https://user-images.githubusercontent.com/46504563/133303865-2025138e-e135-4d06-9a77-61333f5651ce.png)
Since it'll also be helpful if this function can return a string variable in length, I'm going to replace this old method with the new one, and import it in to `HAWCUtils` for use.